sdk/keyvault/azkeys/models.go

```diff
@@ -23,22 +27,34 @@ type Properties struct {
 	// Indicates if the private key can be exported.
 	Exportable *bool `json:"exportable,omitempty"`
 
+	// Key identifier.
+	ID *string
+
+	// READ-ONLY; True if the key's lifetime is managed by key vault. If this is a key backing a certificate, then managed will be true.
+	Managed *bool `json:"managed,omitempty" azure:"ro"`
+
+	Name *string
+
 	// Not before date in UTC.
 	NotBefore *time.Time `json:"nbf,omitempty"`
 
-	// READ-ONLY; Creation time in UTC.
-	CreatedOn *time.Time `json:"created,omitempty" azure:"ro"`
-
 	// READ-ONLY; softDelete data retention days. Value should be >=7 and <=90 when softDelete enabled, otherwise 0.
 	RecoverableDays *int32 `json:"recoverableDays,omitempty" azure:"ro"`
 
 	// READ-ONLY; Reflects the deletion recovery level currently in effect for keys in the current vault. If it contains 'Purgeable'
 	// the key can be permanently deleted by a privileged user; otherwise, only the system
 	// can purge the key, at the end of the retention interval.
-	RecoveryLevel *DeletionRecoveryLevel `json:"recoveryLevel,omitempty" azure:"ro"`
+	RecoveryLevel *string `json:"recoveryLevel,omitempty" azure:"ro"`
+
+	// Application specific metadata in the form of key-value pairs.
+	Tags map[string]string `json:"tags,omitempty"`
 
 	// READ-ONLY; Last updated time in UTC.
 	UpdatedOn *time.Time `json:"updated,omitempty" azure:"ro"`
+
+	VaultURL *string
```

Comment:
Missing doc comments

Reply:
New question: does this go over the wire as part of a request or response body?

Reply:
> @chlowell the VaultURL does not go over the wire, nor is it returned by the service, it's parsed from the Secret ID

What's it for? Why is it a pointer?

Reply:
It can be used for debugging. It is a consistency item with other languages as well. I made it a pointer to be consistent with all the other strings on this model. If the service call returns an error this will be nil and the user couldn't use the VaultURL to do anything like create a new Client (if they were to skip checking the error).